src/bots/comp/embed.js

```diff
@@ -0,0 +1,94 @@
+import Discord from 'discord.js';
+// const embed = new Discord.MessageEmbed()
+// .setColor('#0099ff')
+// .setTitle('Salary Finder Menu')
+// .setAuthor('The official CS Careers Hub Chat Bot')
+// .setDescription('Thank you for requesting salary information from CSCH Chat Bot! We\'re glad to provide you with whatever information you need. \n\n Enter `++salary [value(1)] .. [value(n)]` to chain the following variables:')
+// .addFields(
+//   { name: 'Company', value: 'eg. FB, Apple, MSFT, Google', inline: true },
+//   { name: 'Location', value: 'eg. NYC, Seattle, SF', inline: true },
+//   { name: 'Level', value: 'eg. L3, S4, etc.', inline: true },
+//   { name: 'Focus', value: 'eg. Full-stack, ML, Back-End', inline: true },
+// )
+// .addFields(
+//   { name: 'Example:', value: '`++salary nyc google l3 full-stack`' },
+// )
+
+let compEmbed = {
+
+  compensationFields: [
+    {
+      example: 'Mozilla',
+      name: 'Company',
+    },
+    {
+      example: 'Software Engineer I',
+      name: 'Title',
+    },
+    {
+      example: 'software Engineer',
+      name: 'Date of Offer',
+    },
+    {
+      example: 'Mountain View, California',
+      name: 'Location',
+    },
+    {
+      example: '$70,000',
+      name: 'Salary',
+    },
+    {
+      example: 'None',
+      name: 'Stock/Recurring Bonus',
+    },
+    {
+      example: '$5,000',
+      name: 'Relocation/Signing Bonus',
+    },
+    {
+      example: '80k/year',
+      name: 'Total Compensation',
+    },
+  ],
+  compensationFormatter: function (fields, isExample) {
```

Comment:
We've been using ES6 arrow notation elsewhere

Reply:
I believe there are issues with using 'this' with an arrow function, so I try to steer clear of that.

Reply:
Oh yeah, it does look like it can get a little [complicated](https://www.codementor.io/@dariogarciamoya/understanding-this-in-javascript-with-arrow-functions-gcpjwfyuc) in certain cases. If this code doesn't work with arrow functions, then that's a plenty good reason not to use them :)